src/signers/LightwalletSigner.ts

```diff
@@ -105,6 +114,9 @@ export class LightwalletSigner implements TxSigner {
    * Returns ETH balance of loaded user by querying the relay server.
    */
   public async getBalance (): Promise<Amount> {
+    if (!this._isUserLoaded()) {
+      return Promise.reject(this._userNotLoadedError)
```

Comment:
here I would use throw, as this is one of the reasons why you want to use async/await so that you can write it as usual.

Reply:
Rule of thumb, if you dont have reject, but need too use Promise.reject, you can probably do it just with throw.

Reply:
ah yes makes sense.. good rule of thumb will remember it